src/types/IJaaSMeetingProps.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import IJitsiMeetingOptions from "./IJitsiMeetingOptions";
+import { JaaSMeeting } from "..";
+
+/**
+ * The type of the React {@code Component} props of {@link JaaSMeeting}.
+ */
+export default interface IJaaSMeetingProps {
+  /**
+   * The id of the component
+   */
+  id: string;
+
+  /**
+   * The App ID that provides an isolated context and prefixes the room name
+   */
+  appId: string;
```

Comment:
Does this map to the tenant? How do we refer to this in the JaaS docs?

Reply:
It does, but I'll remove the tenant prop as per the other threads.